wsba-participant-completion-simple/src/test/java/org/jboss/as/quickstarts/wsba/participantcompletion/simple/Client.java

```diff
@@ -57,6 +58,7 @@ public Client() throws MalformedURLException {
          * Add client handler chain so that XTS can add the transaction context to the SOAP messages.
          */
         BindingProvider bindingProvider = (BindingProvider) set;
+        @SuppressWarnings("rawtypes")
```

Comment:
I tried too and gave up after a while. If you think of a solution later, feel free to update the code.